### PR TITLE
TicketingFacilityEnumeration to ScopeOfTicketEnumeration

### LINKED ISSUE #235: ScopeOfTicketListOfEnumerations - wrong item type
Location: https://github.com/NeTEx-CEN/NeTEx/blob/master/xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd

Element: 
<xsd:simpleType name="ScopeOfTicketListOfEnumerations">
    <xsd:annotation>
        <xsd:documentation>List of values for SCOPEs of TICKET.</xsd:documentation>
    </xsd:annotation>
    <xsd:list itemType="**TicketingFacilityEnumeration**"/> --> should be "**ScopeOfTicketEnumeration**"
</xsd:simpleType>

### PR DESCRIPTION
**Pull request by skinkie**
Fix #235

### CODE REVIEW

**Review by Aurige — APPROVED** [review]

**Review by syversenkr — CHANGES_REQUESTED** [review]
1) Not backwards compatible, obviously, but being a bug this fix is necessary. Do we need a deprecation/grace period?

2) I believe the fix on line 1289 should be reverted, because
	<xsd:element name="TicketingFacility" type="TicketingFacilityEnumeration" default="unknown">
should **remain** of TicketingFacilityEnumeration, as this is the "Types of TICKETING available" (i.e. "Allowed values for TicketingFacility")

However, on line line 1277 the ScopeOfTicketList should be of ScopeOfTicketListOfEnumerations type (**not** "TicketingFacilityListOfEnumerations")
I.e. change  to:
	<xsd:element name="ScopeOfTicketList" type="ScopeOfTicketListOfEnumerations">

**Review by ue71603 — APPROVED** [review]

File: xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd [hunk 1]
```diff
@@ -1283,10 +1283,10 @@ Rail transport, Roads and Road transport
 		<xsd:annotation>
 			<xsd:documentation>List of values for SCOPEs of TICKET.</xsd:documentation>
 		</xsd:annotation>
-		<xsd:list itemType="TicketingFacilityEnumeration"/>
+		<xsd:list itemType="ScopeOfTicketEnumeration"/>
 	</xsd:simpleType>
 	<!-- ===Ticketing FACILITY====================================== -->
-	<xsd:element name="TicketingFacility" type="TicketingFacilityEnumeration" default="unknown">
+	<xsd:element name="TicketingFacility" type="ScopeOfTicketEnumeration" default="unknown">
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd:1289]:
```suggestion
	<xsd:element name="TicketingFacility" type="TicketingFacilityEnumeration" default="unknown">
```

File: xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd [hunk 2]
```diff
@@ -1300,7 +1300,7 @@ Rail transport, Roads and Road transport
 		<xsd:annotation>
 			<xsd:documentation>List of values for TICKETING FACILITies.</xsd:documentation>
 		</xsd:annotation>
-		<xsd:list itemType="TicketingFacilityEnumeration"/>
+		<xsd:list itemType="ScopeOfTicketEnumeration"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd:1303]:
```suggestion
		<xsd:list itemType="TicketingFacilityListOfEnumerations"/>
```

File: xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd [hunk 3]
```diff
@@ -1300,7 +1300,7 @@ Rail transport, Roads and Road transport
 		<xsd:annotation>
 			<xsd:documentation>List of values for TICKETING FACILITies.</xsd:documentation>
 		</xsd:annotation>
-		<xsd:list itemType="TicketingFacilityEnumeration"/>
+		<xsd:list itemType="TicketingFacilityListOfEnumerations"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd:1303]:
```suggestion
		<xsd:list itemType="TicketingFacilityEnumerations"/>
```

File: xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd [hunk 4]
```diff
@@ -1300,7 +1300,7 @@ Rail transport, Roads and Road transport
 		<xsd:annotation>
 			<xsd:documentation>List of values for TICKETING FACILITies.</xsd:documentation>
 		</xsd:annotation>
-		<xsd:list itemType="TicketingFacilityEnumeration"/>
+		<xsd:list itemType="TicketingFacilityEnumerations"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_facility_version.xsd:1303]:
```suggestion
		<xsd:list itemType="TicketingFacilityEnumeration"/>
```

**Pull request merged**